app/[localeCode]/tag/[tag]/page.tsx

```diff
@@ -0,0 +1,42 @@
+import type { Locale } from '@prezly/theme-kit-nextjs';
+import type { Metadata } from 'next';
+
+import { app, generatePageMetadata, routing } from '@/adapters/server';
+import { Tag as TagIndexPage } from '@/modules/Tag';
+import { getStoryListPageSize, parsePreviewSearchParams } from 'utils';
+
+interface Props {
+    params: {
+        localeCode: Locale.Code;
+        tag: NonNullable<string>;
+    };
+    searchParams: Record<string, string>;
+}
+
+export async function generateMetadata({ params }: Props): Promise<Metadata> {
+    const { generateUrl } = await routing();
+    const { localeCode, tag } = params;
+
+    return generatePageMetadata({
+        locale: localeCode,
+        title: tag,
```

Comment:
Are tags capitalized? If not, maybe it makes sense to capitalize first letter?

Reply:
https://github.com/prezly/theme-nextjs-bea/pull/1199/commits/139b5919829c5d329e2b308e6b9ee33aea18f9c9